### PR TITLE
Complete the about-me section

### PR DESCRIPTION
**Pull request by KenbonTN**
- [ ] No linter error.
- [ ] README file is more descriptive.
- [ ] I used CSS Grid and Flexbox combined.
- [ ] Implemented the button interactions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+                    </h2>
+                    <section class="card-description">
+                        <ul class="font-style">
+                            <li class="card-list-title on-small"> CANOPY</li>
+                            <li class="on-large card-list-title ">FACEBOOK</li>
+                            <li class="on-large Job-title_and_year">Full Stack Dev</li>
+                            <li class="Job-title_and_year on-small">
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year">
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly 😊 consider the container of the works section is a section tag and for each project using a div tag.

File: index.html [hunk 2]
```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+            </section>
+            <section class="card second-card">
+                <img class="snapshoot-style" src="images/Snapshoot Portfolio2.svg" alt="Snapshoot portfolio image">
+                <section>
+                    <h2 class="card-header font-style on-small">
+                        Multi-Post <br> Stories
+                    </h2>
+                    <h2 class="card-header font-style on-large">
+                        Uber Navigation
+                    </h2>
```
Comment (osamaashraf6) [inline index.html]: - [ ] For each 😃 project there is only on h2 tag not two h2 tags

File: index.html [hunk 3]
```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+                    <p class="notice font-style">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programming-languages font-style">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id="2" class="project-button button" type="button">See Project</button>
+                </section>
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly 👌☺️ fix the design for each project and make sure to match your design with the Figma design and use appropriate padding and margin.

| Figma design | Your design| 
| --- | --- |
| image |  image  |

File: index.html [hunk 4]
```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" fill="currentColor"
+                                    class="bi bi-linkedin icon" viewBox="0 0 16 16">
+                                    <path
+                                        d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z" />
+                                </svg>
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly 💫 use the correct icon for LinkedIn like the Figma design, you can export it from Figma and use it 

| Figma design | Your design| 
| --- | --- |
|   image   |   image    |

File: styles/mobile-first-style.css [hunk 5]
```diff
@@ -167,3 +167,350 @@ section {
...
+.btn:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.languages {
+  margin-top: 30px;
+  margin-left: 24px;
+}
```
Comment (osamaashraf6) [inline styles/mobile-first-style.css]: - [ ] Kindly use the appropriate margin like in the design in the Figma template🤞

| Figma design | Your design| 
| --- | --- |
|   image   |   image    |

File: index.html [hunk 6]
```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+                        </li>
+                    </ul>
+                    <button type="button" class="resume-button button">Get my resume</button>
+                </section>
+            </section>
+            <div class="languages_for_web">
+                <section class="languages">
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly consider the technology categories ("Languages",  💯"Frameworks" and "Skills") are in a `ul`, each of them in a `li`

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @KenbonTN 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Not linter errors present
- Good descriptive pull request

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -52,7 +52,7 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
     
                                 <li>
                                     <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
-                                        <img src="/images/Linkedin icon.svg" alt="Linkedin icon">
+                                        <img src="images/Linkedin icon.svg" alt="Linkedin icon">
                                     </a>
```
Comment (algerina) [inline index.html:56]: - [ Optional] Your Linkedin icon is not displayed correctly kindly, replace the link with the following `Linkedin icon`

**Review by algerina — APPROVED** [review]
Hi @KenbonTN ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
You did great implementing the previews TSE's requirements :+1: 
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**